setup.cfg

```diff
@@ -14,10 +14,10 @@ show-response = 1
 
 [pytest]
 minversion = 2.8
-norecursedirs = ".tox" "build" "docs[\/]_build" "docs[\/]generated" "astropy[\/]extern" "astropy[\/]utils[\/]compat[\/]futures"
+norecursedirs = ".*" "*.egg-info" "build" "docs[\/]_build" "docs[\/]generated" "astropy[\/]extern" "astropy[\/]utils[\/]compat[\/]futures" "astropy_helpers"
```

Comment:
This could probably be simplified using [testpaths](https://docs.pytest.org/en/latest/customize.html#confval-testpaths) (requires pytest 2.8+, which probably explains why it was not used before).